.github/workflows/lint.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: Linters
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: ['**']
+  pull_request:
+    branches: ['main']
+
+  workflow_dispatch:
+
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install yamllint
+        run: sudo apt-get install yamllint
```

Comment:
Have you checked if there are existing actions for yaml linting? There has to be something out there.

Reply:
I wrote this one myself, it was my first Github action, and was based on GitHub's helloworld action.
I'm happy to use something else, I'm sure there are better linters.
I'll take a look.

Reply:
Okay, so I've already moved the YAML lint stuff out to another PR, so by now this is a moot topic. However, just to add closure to the thread, I did look again and I found several GitHub actions that implement YAML linting... just like mine.
https://github.com/marketplace?type=actions&query=yamllint

Moving this conversation to that PR, please.
https://github.com/release-service/release-bundles/pull/2